geomesa-tools/README.md

```diff
@@ -931,3 +931,174 @@ Prints out the version, git branch, and commit ID that the tools were built with
 
 #### Example commands:
     geomesa version
+
+## GeoMesa Kafka Command Line Tools
+GeoMesa also comes with Kafka command line tools which you can run with:
+
+    geomesa-kafka
+    
+This should print out the following usage text: 
+    
+    $ geomesa-kafka
+    Usage: geomesa-kafka [command] [command options]
+      Commands:
+        create          Create a feature definition in GeoMesa
+        describe        Describe the attributes of a given feature in GeoMesa
+        help            Show help
+        list            List GeoMesa features for a given zkPath
+        listen          Listen to a GeoMesa Kafka topic
+        removeschema    Remove a schema and associated features from GeoMesa
+        version         GeoMesa Version
+
+### create
+Used to create a feature type `SimpleFeatureType` at the specified zkpath.
```

Comment:
doc and readme should perhaps indicate that the kafka topic will be created by default...is there a situation where it already exists by chance?